Multisig/UI/Transactions/TransactionDetails/LoadingTransactionDetailsViewModel.swift

```diff
@@ -60,51 +60,43 @@ class TransactionDetailsViewModel: BasicLoadableViewModel {
                 .sink(receiveCompletion: { [weak self] completion in
```

Comment:
per Richard's and Jose's answers, let's add the `delay(3 seconds)` before receiving the message on RunLoop.main so that we'll request the next message after the delay

Reply:
ok